lint/linter/test-versions.ts

```diff
@@ -78,7 +78,7 @@ const isValidVersion = (
     if (version === 'preview') {
       return !!browsers[browser].preview_name;
     }
-    return Object.hasOwn(browsers[browser].releases, version.replace('≤', ''));
+    return !!findBrowserRelease(browsers, browser, version.replace('≤', ''));
```

Comment:
Nit: `findBrowserRelease` already strips `≤`.

```suggestion
    return !!findBrowserRelease(browsers, browser, version);
```